warehouse/templates/packaging/detail.html

```diff
@@ -80,7 +80,16 @@ <h3 class="title"><a href="{{ request.route_path('packaging.project', name=relea
     <p class="help-link"><a href="#" {{ l20n("howToInstall") }}>How do I install this?</a></p>
   </div>
 </section>
-
+{% if release.insecure %}
+<section class="horizontal-section -bad -medium">
+  <div class="site-container">
+    <p {{ l20n("versionInsecure") }}>
+            This version has been marked insecure by the package maintainers. You can read more
+            about the vulnerabilites <a href="{{ release.insecure_url }}">here</a>.
```

Comment:
On number 2:
You could either do what @alexwlchan suggests, or simply make the whole sentence a link:
`<a href="{{ release.insecure_url }}">You can read more about the vulnerabilities here</a>`